test/unit/v1.0.0/normalize-v1.test.js

```diff
@@ -1282,6 +1282,176 @@ describe('v1.0.0 normalization', function () {
+                                method: 'POST',
+                                data: [
+                                    {
+                                        key: 'file',
+                                        value: null,
+                                        type: 'file'
+                                    }
+                                ],
+                                dataMode: 'params',
+                                collectionId: '84b2b626-d3a6-0f31-c7a0-47733c01d0c2'
+                            }
+                        ]
+                    });
+                    done();
+                });
+            });
+
+            it('should correctly handle non-string file entities in requests', function (done) {
+                transformer.normalizeSingle({
+                    id: '4f65e265-dd38-0a67-71a5-d9dd50fa37a1',
+                    headers: '',
+                    url: 'https://postman-echo.com/post',
+                    method: 'POST',
+                    data: [
+                        {
+                            key: 'file',
+                            value: [],
+                            type: 'file'
+                        }
+                    ],
+                    dataMode: 'params',
+                    collectionId: '84b2b626-d3a6-0f31-c7a0-47733c01d0c2'
+                }, options, function (err, result) {
+                    expect(err).not.to.be.ok;
+
+                    expect(JSON.parse(JSON.stringify(result))).to.eql({
+                        id: '4f65e265-dd38-0a67-71a5-d9dd50fa37a1',
+                        headers: '',
+                        url: 'https://postman-echo.com/post',
+                        method: 'POST',
+                        data: [
+                            {
+                                key: 'file',
+                                value: null,
+                                type: 'file'
+                            }
+                        ],
+                        dataMode: 'params',
+                        collectionId: '84b2b626-d3a6-0f31-c7a0-47733c01d0c2'
+                    });
+                    done();
+                });
+            });
+
+            it('should set missing file values to null when missing and noDefaults is false', function (done) {
+                transformer.normalizeSingle({
+                    id: '4f65e265-dd38-0a67-71a5-d9dd50fa37a1',
+                    headers: '',
+                    url: 'https://postman-echo.com/post',
+                    method: 'POST',
+                    data: [
+                        {
+                            key: 'file',
+                            type: 'file'
+                        }
+                    ],
+                    dataMode: 'params',
+                    collectionId: '84b2b626-d3a6-0f31-c7a0-47733c01d0c2'
+                }, _.defaults({ noDefaults: false }, options), function (err, result) {
```

Comment:
need to add tests for `noDefaults: true` because false is the default. we could remove the explicit false here as well